### ISSUE TITLE
Can inline search displays more information?

### ISSUE THREAD

**Issue by zwujohnny** [dive2Pro/roam-search-plus#21]
The inline search is very promising,  I can see it helps me to solidifies the search pattern and acts like a kind of backlinks.

The problem is that the display should show more information:
1. the context of the result. Like the format in pop-up mode, I can see the paths for each result, this really helps me to identify which one is useful quickly.
2. highlight the searched keyword. Now when I use the inline method, I find it really hard to parse the result even though they are the notes I wrote before.
3.  other information, eg the page name of the result etc.

Without the above functions, I cannot use it properly and can only stick with the pop-up mode.

**Comment by dive2Pro** [comment]
1 and 3 have been achieved.
I'll think about the second point some more.
